{"name": "test_update_post", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'resources/resource_retrieve_test/payload_create_post.json'", "trace": "@pytest.fixture(autouse=True)\n    def setup_prerequisites():\n        global TOKEN\n        global ID_POST\n    \n        URL = config(\"URL\")\n>       payload = load_json_expected_result(\"resources/resource_retrieve_test/payload_create_post.json\")\n\ntest\\test_update_post.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\npath = 'resources/resource_retrieve_test/payload_create_post.json'\n\n    def load_json_expected_result(path):\n    \n>       with open(path) as file_json:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'resources/resource_retrieve_test/payload_create_post.json'\n\ntest\\test_create_post.py:26: FileNotFoundError"}, "start": 1659906220736, "stop": 1659906220736, "uuid": "7ec1e420-486d-4ecc-ab9c-63061943e6f5", "historyId": "d49df81e7a235110509f054de591a198", "testCaseId": "951045abe6c7bbf3d2846d957a9ea254", "fullName": "test.test_update_post#test_update_post", "labels": [{"name": "suite", "value": "acceptance_testing"}, {"name": "epic", "value": "regression_testing"}, {"name": "epic", "value": "acceptance_testing"}, {"name": "epic", "value": "smoke_testing"}, {"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression_testing"}, {"name": "tag", "value": "smoke_testing"}, {"name": "tag", "value": "acceptance_testing"}, {"name": "parentSuite", "value": "test"}, {"name": "host", "value": "DESKTOP-QQRK36A"}, {"name": "thread", "value": "6124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_update_post"}]}